project3/style.css

```
@@ -0,0 +1,166 @@
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+.clearfix::before,
+.clearfix::after {
+    content: " ";
+    display: table;
+}
+
+.clearfix::after {
+    clear: both;
+}
+
+/* as small as 375, as large as 1280 */
+
+body {
+    background-image: linear-gradient(hsl(56, 41%, 93%), hsl(208, 77%, 90%));
+    border-bottom: 69px solid hsl(203, 48%, 26%);
+    font-family: 'Bitter', serif;
+    margin: 0 0 1% 0;
+}
+
+h1,
+h2 {
+    font-weight: 700;
+}
+
+
+
+h1 {
+    background: hsl(203, 48%, 26%);
+    color: hsl(56, 41%, 93%);
+    font-size: 50px;
+    margin: 1% 0 0 0;
+    padding: 0 0 0 15%;
```

Since an `h1` doesn't have default padding, you can just set the property you care about:
```suggestion
    padding-left: 15%;
```
While this seems like a bit more typing, but I like it b/c it's so specific and clear.